### PR TITLE
Highlight end turn when there are no valid moves left

### PR DESCRIPTION
**Pull request by JoCodesThings**
Fixes Issue #121

### CODE REVIEW

File: client/src/app/game/board/board.tsx [hunk 1]
```diff
@@ -121,6 +118,9 @@ function Board({onClick, localGameState}: Props) {
 		)
 	}
 
+	const noAvailableActions = () =>
```
Comment (Sense101) [inline client/src/app/game/board/board.tsx:121]: Please define this function before it is used.

File: package-lock.json
[the remark this answers is not in the document]
Comment (JoCodesThings) [reply package-lock.json:1]: Gotcha

File: client/src/app/game/board/board.tsx (hunk 1 above)
Comment (JoCodesThings) [reply client/src/app/game/board/board.tsx]: Moved

**Comment by Sense101** [comment]
Before I approve these changes, can I ask why your commits are unverified?

**Comment by JoCodesThings** [comment]
Because I'm a dummy and hadn't set up GPG after accidentally nuking `/home` during a reinstall. I actually just fixed it up.

**Comment by JoCodesThings** [comment]
How about that 🙂

**Review by Sense101 — APPROVED** [review]

**Comment by Sense101** [comment]
Thanks again for the pr! This will go live during the next update. Do you have a discord account which I can credit?

**Comment by JoCodesThings** [comment]
> Thanks again for the pr! This will go live during the next update. Do you have a discord account which I can credit?

Yep, it's Vesenna#9809

**Pull request merged**